packages/GA4Client/src/event-handler.js

```diff
@@ -2,8 +2,16 @@ function EventHandler(common) {
     this.common = common || {};
 }
 
+EventHandler.prototype.eventFunction = function (eventName, eventAttributes) {
```

Comment:
I think this would be more descriptive
```suggestion
EventHandler.prototype.sendEventToGA4 = function (eventName, eventAttributes) {
```